### PR TITLE
fix: Snapshot Persistance

### PR DESCRIPTION
**Pull request by bbernays**

### CODE REVIEW

File: pkg/policy/snapshots.go [hunk 1]
```diff
@@ -207,19 +207,21 @@ func (ce *Executor) checkTableExistence(ctx context.Context, tableName string) (
 	explainQuery := fmt.Sprintf("select pg_get_viewdef('%s'::regclass::oid) ", tableName)
 	rows, err := ce.conn.Query(ctx, explainQuery)
 	if err != nil {
+		ce.log.Error("error running explain", "tableName", tableName)
```
Comment (disq) [inline pkg/policy/snapshots.go:210]: log the `err` as well?

File: pkg/policy/snapshots.go [hunk 2]
```diff
@@ -207,19 +207,21 @@ func (ce *Executor) checkTableExistence(ctx context.Context, tableName string) (
...
 		return "", err
 	}
+	if err := rows.Err(); err != nil {
```
Comment (disq) [inline pkg/policy/snapshots.go:213]: As per doc, `rows.Err` returns errors occured while reading, so moving this before the reading part (which is `Scan`) is not correct? But there's a similar thing in line 152 of the same file where the rows.Err block should be moved to after rows.Next around line 111.

**Review by disq — CHANGES_REQUESTED** [review]

File: pkg/policy/snapshots.go (hunk 2 above)
Comment (bbernays) [reply pkg/policy/snapshots.go]: Thanks for that catch! Fixed it

**Review by disq — APPROVED** [review]

**Comment by disq** [comment]
LGTM :)

**Pull request merged**